sound/soc/intel/boards/sof_da7219_max98373.c

```diff
@@ -303,8 +331,8 @@ static struct snd_soc_dai_link dais[] = {
 	},
 };
 
-static struct snd_soc_card card_da7219_m98373 = {
-	.name = "da7219max",
+static struct snd_soc_card card_da7219_max = {
+	.name = "da7219max98360a",
```

Comment:
please don't do this, this is going to break existing solutions for UCM, etc.

Add a new card if needed.

Reply:
Ack, indeed, will fix in next drop.